### DISCUSSION TITLE
Lab: Descriptive Analysis II

### DISCUSSION THREAD

**Discussion by mbcann01** [brad-cannell/intro.r.public#31] [Labs]
Please feel free to post questions about this lab below. You are also welcome to leave comments below about the usefulness of this lab. Which parts were helpful? What could I do to improve it? What is still unclear?

**Comment by OJONESD** [comment]
Hello Professor Cannell,

I just wanted to comment I appreciate this lab has us make different types of graphs including the side by side histograms, and comparison by group in the same box plot. I definitely see myself coming back to this lab quarto file later to remember how to use these functions again for other projects.

Thank you,
Olivia Jones

**Reply by mbcann01** [reply]
Hi @OJONESD,

Thank you so much for the feedback! I'm glad you found the lab useful!

Respectfully,
Brad
